README.rst

```diff
@@ -68,14 +68,11 @@ Usage
 =====
 
 Because we don't want to require users to share the lock instance across processes you will have to give them names.
-Eg::
 
-    conn = StrictRedis()
-    with redis_lock.Lock(conn, "name-of-the-lock"):
-        print("Got the lock. Doing some work ...")
-        time.sleep(5)
+Locks
```

Comment:
This new heading needs to be above the paragraph or be removed. Somehow the code block needs to be tied to the explanatory paragraph.

Reply:
Done, removed `Locks` heading as it sounds redundant.